content/providers/quick-start.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "title": "Qiskit Stand-alone",
+    "title": "Qiskit (without a provider)",
```

Comment:
this should be updated to `Qiskit (with built-in simulator)`

Reply:
Resolved in https://github.com/Qiskit/qiskit.org/pull/3113/commits/6bc0dbf465f3d04bb6012c1df1a78f1b6e8ca31e